### PR TITLE
Moved Guzzle to development requirements

### PR DESCRIPTION
**Pull request by edgardmessias**
Guzzle is used only for tests.

This reduces the final size of the release. (+- 700KB)

### CODE REVIEW

**Comment by edgardmessias** [comment]
I'm working on it

**Comment by edgardmessias** [comment]
Done.

File: tests/bootstrap.php [hunk 1]
```diff
@@ -38,6 +38,11 @@
 include_once __DIR__ . '/../inc/includes.php';
 include_once __DIR__ . '/DbTestCase.php';
 
+// check folder exists instead of class_exists('\GuzzleHttp\Client'), to prevent global includes
+if(!file_exists(__DIR__ . '/../vendor/guzzlehttp/guzzle')){
```
Comment (remicollet) [inline tests/bootstrap.php]: We need a sniffer for missing space after if ;)

Comment (remicollet) [inline tests/bootstrap.php]: And another for missing space between ) and {

Comment (edgardmessias) [inline tests/bootstrap.php]: Thanks

Comment (edgardmessias) [inline tests/bootstrap.php]: @remicollet, see this rule for phpcs `Squiz.ControlStructures.ControlSignature` to automate it

**Pull request merged**